graylog2-web-interface/src/components/common/Wizard.jsx

```diff
@@ -114,30 +147,40 @@ class Wizard extends React.Component {
             </Button>
           </ButtonToolbar>
         </div>
-        <Nav bsStyle="pills" activeKey={this.state.selectedStep} onSelect={this._wizardChanged}>
-          {this.props.steps.map((navItem) => {
-            return (<NavItem key={navItem.key} eventKey={navItem.key} disabled={navItem.disabled}>{navItem.title}</NavItem>);
+        <Nav bsStyle="pills"
+             className={navigationClassName}
+             activeKey={selectedStep}
+             onSelect={this._wizardChanged}
+             justified={justified}>
+          {steps.map((navItem) => {
+            return (
+              <NavItem key={navItem.key} eventKey={navItem.key} disabled={navItem.disabled}>{navItem.title}</NavItem>);
           })}
         </Nav>
       </Col>
     );
   };
 
+  _getSelectedStep() {
+    const { activeStep } = this.props;
+    const { selectedStep } = this.state;
+    return lodash.defaultTo(activeStep, selectedStep);
+  }
+
   render() {
-    const rightComponentCols = this.props.horizontal ? 5 : 3; // If horizontal, use more space for this component
+    const { steps, horizontal, containerClassName, children } = this.props;
+    const rightComponentCols = horizontal ? 5 : 3; // If horizontal, use more space for this component
     return (
-      <Row className={this.props.containerClassName}>
-        {this.props.horizontal ? this._renderHorizontalStepNav() : this._renderVerticalStepNav()}
+      <Row className={containerClassName}>
+        {horizontal ? this._renderHorizontalStepNav() : this._renderVerticalStepNav()}
         <Col md={7}>
```

Comment:
Not sure if this component should dictate content width with the `col-md-7` class that it generated.

![image](https://user-images.githubusercontent.com/122591/59301504-5e152c00-8c57-11e9-8c40-0c707a62a9b7.png)

Reply:
The original purpose of that was to leave space on the right side to render a preview, which I think is fine. The page you are working on is also the first one using some centered content, making this problem more apparent than if we added some left-side content.

I think we can add a prop to decide if a preview should be rendered or not, and use the full container width if there's no preview. In that way consumers can decide if they want to render a preview on the right, even if that side of the screen is initially empty. Would that solve that issue for you?

Reply:
Well, `children` should be present if there is a preview, so we shouldn't really need a prop for it. Just need to do something similar to line 215 probably.

Reply:
I was thinking in case the initial preview had no content, but you are right, having a preview without content doesn't make much sense. Will take care of that 👍